src/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../../footerlogo.svg';
+import {
+  Wrapper, Ul, Img, Li,
+} from './Footer.style';
+
+const Footer = () => (
+  <Wrapper>
+    <Ul>
+      <Li>
+        <Link to={{ pathname: 'https://ooloo.io/employers' }} target="_blank">
```

Comment:
Didn't know this works as well. Alternatively you can write

```
<Link as="a" href="https://ooloo.io/employers" target="_blank">
```

This way you have a normal anchor tag outside of the react-router excosystem

Reply:
Ah, now I see that the `Link` is not styled. Then simply write

```
<a href="https://ooloo.io/employers" target="_blank">
```

No need to use the `Link` component here

Reply:
sweet ok, I'll make that change. 👍